src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py

```diff
@@ -82,6 +84,10 @@ def create_webapp(resource_group_name, name, plan, runtime=None, startup_file=No
                              deployment_source_branch, deployment_local_git)
 
     _fill_ftp_publishing_url(webapp, resource_group_name, name)
+
+    if linux_storage_default:
```

Comment:
Can you set this as part of `client.web_apps.create_or_update`?
Also please update test cases to incorporate this change, such as the `test_linux_webapp_quick_create`

Reply:
We can have just 1 line ->             `site_config.app_settings.append(NameValuePair('WEBSITES_ENABLE_APP_SERVICE_STORAGE', "false"))` in the `elif` branch

Reply:
the setting can be applied only after the app is created and that was the reason for not having this inside the elif block